sdk/ml/azure-ai-ml/azure/ai/ml/operations/_compute_operations.py

```diff
@@ -416,6 +421,26 @@ def list_sizes(self, *, location: Optional[str] = None, compute_type: Optional[s
             ]
         return [VmSize._from_rest_object(item) for item in size_list.value]
 
+    @distributed_trace
+    @monitor_with_activity(ops_logger, "Compute.enablesso", ActivityType.PUBLICAPI)
+    @experimental
+    def enable_sso(self, *, name: str, enable_sso: bool = True):
```

Comment:
is there a spec for this change? This API feels awkward to me:

`client.compute.enable_sso(enable_sso=True/False)`

The verb `enable_sso` suggest something will always be turned on, but we can actually turn it off using the same `enable_sso` verb.

Reply:
its specifically for data scientist role who don't have write access to compute instance, rest of role can use write/update API to perform same operation.